tutorial_code/example2_2_frustrated_loop_anneal.py

```diff
@@ -209,47 +228,38 @@ def run_experiment(param, shim, stats, embeddings, alpha_Phi=0.0, alpha_J=0.0):
         shim (dict): shimming data
         stats (dict): dict of sampled statistics
         embeddings (List[dict]): list of embeddings
-        alpha_Phi (float, optional): learning rate for linear shims. Defaults to 0..
-        alpha_J (float, optional): learning rate for coupling shims. Defaults to 0..
+        alpha_Phi (float): learning rate for linear shims. Defaults to 0.
+        alpha_J (float): learning rate for coupling shims. Defaults to 0.
```

Comment:
Minor, but staying with the same convention (capitilizing sentences vs. all lower case, and/or punctuation) look better. Cannot suggest changes to unchanged code.